python_modules/dagster/dagster/_core/storage/runs/schema.py

```diff
@@ -45,6 +45,8 @@
     db.Column("start_time", db.Float),
     db.Column("end_time", db.Float),
     db.Column("backfill_id", db.String(255)),
+    db.Column("run_creation_time", db.Float),
```

Comment:
we should add a comment here about how this differs from `create_timestamp` / `update_timestamp`.